jetty-ee8/pom.xml

```diff
@@ -422,292 +422,4 @@
-                        <groupId>jakarta.servlet.jsp.jstl</groupId>
-                        <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!1.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee8, we need to stay on 1.x of transaction spec
-                             ee9 starts with version 2.x -->
-                        <groupId>jakarta.transaction</groupId>
-                        <artifactId>jakarta.transaction-api</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!1.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee8, we need to stay on 2.1.x of apache directory api+impl (LDAP) -->
-                        <groupId>org.apache.directory.api</groupId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!2.1.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee8, we need to stay on 9.0.x of mortbay apache-jsp impl
-                             ee10 starts with version 10.0.x
-                             ee11 TODO -->
-                        <groupId>org.mortbay.jasper</groupId>
-                        <artifactId>mortbay-apache-jsp</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!9.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- ignore unstable mina releases -->
-                        <groupId>org.apache.mina</groupId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>.+-M[0-9]+$</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                    </rules>
-                  </ruleSet>
-                </configuration>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-  </profiles>
```

Comment:
Lets hold off on cleaning up the pom right now.
We can remove this section later, once we've confirmed the new dependabot configs.